CHANGELOG.md

```diff
@@ -1,6 +1,46 @@
 # Change Log for rocWMMA
 
-Full documentation for rocWMMA is available at [rocwmma.readthedocs.io](https://rocwmma.readthedocs.io/en/latest/).
+Full documentation for rocWMMA is available at [rocm.docs.amd.com/projects/rocWMMA](https://rocm.docs.amd.com/projects/rocWMMA/en/latest).
```

Comment:
Back to docs.amd.com?

Reply:
Yes, this is the one that needs to be used